src/ConfigPage/ConfigPage.css

```
@@ -1,3 +1,22 @@
 .config-page .btn-update-token {
 	margin-top: 1rem;
 }
+
+.config-page {
+	align-items: center;
+	align-content: center;
```

I always try to keep my CSS style rules in alphabetic order, so I am going to rewrite these two lines as the following -

```suggestion
	align-content: center;
	align-items: center;
```